### ISSUE TITLE
jacobian with tuple arg_num crashes

### ISSUE THREAD

**Issue by eric-wieser** [HIPS/autograd#542]
Minimal example:

```python
import autograd.numpy as np
from autograd import jacobian

def my_add(a, b):
    return a + b

a0 = np.array([1.0, 2.0])
b0 = np.array([3.0, 4.0])
my_add_j = jacobian(my_add, argnum=(0, 1))
my_add_j(a0, b0)
# TypeError: 'ArrayVSpace' object cannot be interpreted as an integer
```

**Comment by j-towns** [comment]
Yep, unfortunately Jacobian is only implemented for integer argnum and functions which output a single array (not a tuple). We should update the docstring to reflect this.

FYI the [Jacobian](https://github.com/google/jax/blob/master/jax/api.py#L425) [operators](https://github.com/google/jax/blob/master/jax/api.py#L470) in JAX are more flexible and do support taking the Jacobian w.r.t. multiple arguments, as well as (nested) tuple, list and dict arguments and functions which output (nested) tuples, lists or dicts.
